### DISCUSSION TITLE
New field in report from begin date`

### DISCUSSION THREAD

**Discussion by rel4xdev** [kimai/kimai#2605] [Questions & Answers]
Hi, I want generate new filed from begin date and show this field in report section of timesheet. (just for show another format of date, I dont want save any thing in DB)
Is there any event that I use that for this purpose ?

**Comment by kevinpapst** [comment]
> report section of timesheet

What is that?

**Reply by rel4xdev** [reply]
[https://demo-stable.kimai.org/en/team/timesheet/](https://demo-stable.kimai.org/en/team/timesheet/)

**Reply by rel4xdev** [reply]
Any idea ???

**Comment by rel4xdev** [comment]
I just want generate new filed from begin date and show this field in data table
